pubsub/mqtt3/metadata.yaml

```diff
@@ -0,0 +1,67 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: mqtt3
+version: v1
+status: stable
+title: "MQTT3"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-mqtt3/
+authenticationProfiles:
+  - title: "Connection string"
+    description: "The MQTT3 Connection/Authentication."
+    metadata:
+      - name: url
+        type: string
+        required: true
+        sensitive: true
+        description: |
+          Address of the MQTT broker.
+          Use the tcp:// URI scheme for non-TLS communication. 
+          Use the ssl:// URI scheme for TLS communication.
+        example: '"tcp://[username][:password]@host.domain[:port]"'
+metadata:
+  - name: consumerID
```

Comment:
This is a built-in property for all pubsub components, so no need to add it here. See .build-tools/pkg/metadataschema/validators.go